packages/block-library/src/file/save.js

```diff
@@ -25,7 +25,9 @@ export default function save( { attributes } ) {
 		previewHeight,
 	} = attributes;
 
-	const pdfEmbedLabel = RichText.isEmpty( fileName ) ? 'PDF embed' : fileName;
+	const pdfEmbedLabel = RichText.isEmpty( fileName )
+		? 'PDF embed'
+		: fileName.toPlainText();
```

Comment:
So we need to decide here whether we revert this to `toString` for now (and keep trunk's behaviour), or address #56710, which would be needed to ensure a rich text value in the save function.

Reply:
you mentioned it was already an existing bug, so we don't have to _also_ fix it in this PR do we? revert to trunk's behavior?

Reply:
Ok, reverted it for now.